src/Extensions/SearchFormFactoryExtension.php

```diff
@@ -22,7 +22,14 @@ public function updateForm(
         $fields = $form->Fields()->findOrMakeTab('Editor.Details');
         $file = $context['Record'] ?? null;
 
-        if (!$fields || !$file || $file instanceof Image) {
+        $excludedClasses = Config::inst()->get(self::class, 'exclude_classes') ?? [];
```

Comment:
this is excellent - would you be able to add a default config yml to exclude images? I think that would be expected behaviour

Reply:
Done